### PR TITLE
Update to use the fetchImageToLabel Cloud Function

### PR DESCRIPTION
**Pull request by marcosjordao**
Depends on #17

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "build": "sh -ac '. ./.env.${REACT_APP_ENV}; react-scripts build'",
     "build:test": "REACT_APP_ENV=test yarn build",
     "build:production": "REACT_APP_ENV=production yarn build",
-    "test": "firebase emulators:exec --project aiscope-labelling-app-test --import mockData 'react-scripts test --watchAll=false --detectOpenHandles --forceExit'",
+    "test": "react-scripts test --watchAll=false --detectOpenHandles --forceExit",
+    "test:emulator": "firebase emulators:exec --project aiscope-labelling-app-test --import mockData 'react-scripts test --watchAll=false --detectOpenHandles --forceExit'",
```
Comment (sendamiluque7013) [inline package.json:12]: These tests are not going to be executed by the pipeline, is this expected?

File: src/services/ImagesService/ImagesService.test.ts [hunk 2]
```diff
@@ -24,65 +22,27 @@ describe('ImagesService', () => {
...
+      sampleLocation: '1',
+      sampleReference: '1',
+      masks: [],
+      labellers: [],
+      markedAsInvalid: 0,
+      isCompleted: false,
+      createdOn: new Date(),
+    }
+
+    it('should call the fetchImageToLabel cloud function and return the image ', async () => {
```
Comment (sendamiluque7013) [inline src/services/ImagesService/ImagesService.test.ts:38]: I am missing the tests when there is no image returned

**Review by sendamiluque7013 — CHANGES_REQUESTED** [review]

File: src/services/ImagesService/ImagesService.test.ts (hunk 2 above)
Comment (marcosjordao) [reply src/services/ImagesService/ImagesService.test.ts:38]: Added

File: package.json (hunk 1 above)
Comment (marcosjordao) [reply package.json:12]: Yes, it is expected.

The tests with firebase emulator are considerable expensive to run (in terms of time) and once the `DatabaseService` has been removed there is no need to run the tests using the emulator.

To keep the possibility and the approach to run the tests with the emulator I decided to keep it as a "secondary" run script (`test:emulator`).

**Review by sendamiluque7013 — APPROVED** [review]

**Pull request merged**